### ISSUE TITLE
[Bug] Unable to open 'show.blade.php', Actually, the file(resources/views/center/rules/show.blade.php ) exists.

### ISSUE THREAD

**Issue by faydanube** [codingyu/laravel-goto-view#76]
```php
return view('center.rules.show', compact('list'));
```

Click `center.rules.show`

Alert

<img width="349" alt="image" src="https://github.com/user-attachments/assets/6619a977-6c82-4e9e-b867-53a7faedfc26" />

Click `Create`, Alert

<img width="311" alt="image" src="https://github.com/user-attachments/assets/46933219-f5a3-4679-ae8b-7cc98bbbf42b" />

Actually, the file(`resources/views/center/rules/show.blade.php
`) exists.

**Comment by codingyu** [comment]
project in the workspace?

**Comment by faydanube** [comment]
> project in the workspace?

Yes, all operations are the same as normal development and can be created normally after uninstalling the extension. I'll record a gif next time I revive it, or maybe it's been fixed in the meantime?
